from ..exceptions import (
    CallbackSuccess as CallbackSuccess,
    FeasibleSuccess as FeasibleSuccess,
    MaxEvalError as MaxEvalError,
    TargetSuccess as TargetSuccess,
)

def test_max_eval_error(): ...
def test_target_success(): ...
def test_callback_success(): ...
def test_feasible_success(): ...
